.github/workflows/connected-test.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run Connected Tests
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ '**' ]
+jobs:
+  connected-test:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
```

Comment:
Just FYI - there's a PR (currently in review) that upgrades the postgres version from `9.6` to `12.3` https://github.com/DataBiosphere/terra-workspace-manager/pull/134

Reply:
Thanks for the heads up. @davidangb I think I might merge before you, so please update this file as well.